packages/flutter/lib/src/widgets/modal_barrier.dart

```diff
@@ -54,6 +55,16 @@ class ModalBarrier extends StatelessWidget {
   ///    [ModalBarrier] built by [ModalRoute] pages.
   final bool dismissible;
```

Comment:
This doc probably also needs updating (or at least link to onDismiss).

Reply:
Done. Thx.